core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/workspace/WorkspaceResourceMonitor.scala

```diff
@@ -92,7 +92,7 @@ class WorkspaceResourceMonitor(
     jobResults <- Future.sequence(
       registeredRunners
         .getOrElse(job.jobType, List())
-        .map(_.run(job).recover { case t =>
+        .map(_.run(job).map(_.complete).recover { case t =>
```

Comment:
What does this do?

Reply:
@blakery still wondering about this one

Reply:
I changed the return type of the job runner, to be a _wrapped_ value of a boolean:
```scala
  implicit sealed class JobStatus(val complete: Boolean)

  case object Complete extends JobStatus(true)

  case object Incomplete extends JobStatus(false)
```
This means that the return type will either be `Complete` or `Incomplete`, which has 2 advantages:
1. It reads a bit more clearly in the job runner itself - instead of returning raw booleans everywhere, you're returning a type that makes it clear what it indicates
2. Because it's a wrapped type, the handling has to be a little bit more explicit, which reduces the chance of doing something unintended

The change here is a result of 2